### ISSUE TITLE
Gnuplot does not pickup Quicktime installation

### ISSUE THREAD

**Issue by davydden** [spack/spack#9721]
I was trying to build `gnuplot` with `quick-time` and it failed due to missing headers.

```
>> 443    qtterminal/qt_term.cpp:51:10: fatal error: 'QtCore' file not found
     444    #include <QtCore>
     445             ^~~~~~~~
```

If I check flags then I see 
```
<QTPREFIX>/include/QtCore
<QTPREFIX>/include/QtGui
<QTPREFIX>/include/QtSvg
<QTPREFIX>/include/QtWidgets
<QTPREFIX>/include/QtPrintSupport
```

I see none of those in Spack's installed QT on macOS:
```
QtAccessibilitySupport		QtDeviceDiscoverySupport	QtEventDispatcherSupport	QtFontDatabaseSupport		QtOpenGLExtensions		QtPlatformCompositorSupport	QtQmlDebug			QtThemeSupport
QtClipboardSupport		QtEdidSupport			QtFbSupport			QtGraphicsSupport		QtPacketProtocol		QtPlatformHeaders		QtServiceSupport		QtUiTools
```

### Information on your system

 1. macOS Mojave 10.14.1

p.s. looking at Homebrew's [qt](https://github.com/Homebrew/homebrew-core/blob/master/Formula/qt.rb), [gnuplot](https://github.com/Homebrew/homebrew-core/blob/master/Formula/gnuplot.rb) or Macports [gnuplot](https://github.com/macports/macports-ports/blob/master/math/gnuplot/Portfile), I don't see them doing anything special.

**Comment by davydden** [comment]
hm, looks like it's Gnuplot's issue, i see
```
<QTPREFIX>/lib/QtCore.framework
<QTPREFIX>/lib/QtGui.framework
<QTPREFIX>/lib/QtSvg.framework
<QTPREFIX>/lib/QtWidgets.framework
<QTPREFIX>/lib/QtPrintSupport.framework
```

**Comment by davydden** [comment]
fixed in #9722
